hack/check_upgrade_ovs.sh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
+#
+# This file is part of the KubeVirt project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Copyright 2020 Red Hat, Inc.
+#
+
+set -ex
+
+if [[ ${RELEASE_DELTA} == 2 ]]; then
+  # upgrading from the previous minor version
+  if [[ -z ${PREVIOUS_OVS_ANNOTATION} ]] || [[ ${PREVIOUS_OVS_ANNOTATION} == 'true' ]]; then
+    # if the annotation is "true" or not existed in previous version - OVS should be deployed in the new version.
+    echo "check that OVS annotation in HCO CR in new version is set to true"
+    [[ $(${CMD} get HyperConverged kubevirt-hyperconverged -n kubevirt-hyperconverged -o jsonpath='{.metadata.annotations.deployOVS}') == 'true' ]]
+
+    echo "check that OVS exists in CNAO CR Spec"
+    [[ $(${CMD} get networkaddonsconfigs cluster -o jsonpath='{.spec.ovs}') == '{}' ]]
+
+    echo "check that OVS DaemonSet exists"
+    [[ $(${CMD} get ds ovs-cni-amd64 -n kubevirt-hyperconverged --no-headers --ignore-not-found | wc -l) == '1' ]]
+
+  elif [[ -n ${PREVIOUS_OVS_ANNOTATION} ]] && [[ ${PREVIOUS_OVS_ANNOTATION} != 'true' ]]; then
```

Comment:
continuing the discussion from [here](https://github.com/kubevirt/hyperconverged-cluster-operator/pull/992#discussion_r542387935), I meant you can remove this entire condition since it'll always happen.
```suggestion
  elif
```

Reply:
you mean just `else`?
you're right that the two conditions are complimentary.